inc/customizer/controls/layouts.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+class Primer_Customizer_Layouts_Control extends WP_Customize_Control {
+
+	/**
+	 * Enqueue some custom css for this control
+	 */
+	public function enqueue() {
+
+		$rtl    = is_rtl() ? '-rtl' : '';
+		$suffix = SCRIPT_DEBUG ? '' : '.min';
+
+		wp_enqueue_style( 'primer-layouts', get_template_directory_uri() . "/assets/css/admin/layouts{$rtl}{$suffix}.css", array(), PRIMER_VERSION );
+
+	}
+
+	/**
+	 * Override default template with a custom one
+	 */
+	protected function render_content() {
+
+		global $primer_customizer_layouts;
+
+		if ( ! empty( $this->label ) ) :?>
+
+			<span class="customize-control-title"><?php echo esc_html( $this->label ); ?></span>
+
+		<?php endif;
+
+		if ( ! empty( $this->description ) ) : ?>
+
+			<span class="description customize-control-description"><?php echo $this->description ; ?></span>
```

Comment:
Typically we don't use delimiters on one-liners, also there is a rouge space in this one: `$this->description ; ?>`